python/beeai_framework/agents/react/events.py

```diff
@@ -90,7 +90,4 @@ class ReActAgentToolEvent(BaseModel):
     "success": ReActAgentSuccessEvent,
     "update": ReActAgentUpdateEvent,
     "partial_update": ReActAgentUpdateEvent,
-    "tool_start": ReActAgentToolEvent,
-    "tool_success": ReActAgentToolEvent,
-    "tool_error": ReActAgentToolEvent,
```

Comment:
They are not being used

Reply:
is `ReActAgentToolEvent` being used anywhere else? if not, should it also be removed?

Reply:
as for `tool_start` and `tool_success` they are used in TS, but are missing from our python implementation still: https://github.com/i-am-bee/beeai-framework/blob/a28c9046efecaab9006839368c7114c488df9ea3/typescript/src/agents/react/runners/default/runner.ts#L204 and https://github.com/i-am-bee/beeai-framework/blob/a28c9046efecaab9006839368c7114c488df9ea3/typescript/src/agents/react/runners/default/runner.ts#L216

Reply:
We don't want to use them. One can listen to tool calls with nested events.